Api/Settings.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace Drip\Connect\Api;
+use Drip\Connect\Api\SettingsInterface;
+
+class Settings implements SettingsInterface
+{
+    /** @var \Drip\Connect\Model\ConfigurationFactory */
+    protected $configFactory;
+
+    /** @var \Magento\Store\Model\StoreManagerInterface */
+    protected $storeManager;
+
+    /** @var \Magento\Framework\App\ProductMetadata */
+    protected $productMetadata;
+
+    /** @var \Magento\Framework\Module\ResourceInterface */
+    protected $moduleResource;
+
+    public function __construct(
+        \Drip\Connect\Model\ConfigurationFactory $configFactory,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
+        \Magento\Framework\App\ProductMetadata $productMetadata,
+        \Magento\Framework\Module\ResourceInterface $moduleResource
+    ) {
+        $this->configFactory = $configFactory;
+        $this->storeManager = $storeManager;
+        $this->productMetadata = $productMetadata;
+        $this->moduleResource = $moduleResource;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function updateSettings($websiteId, $accountParam, $integrationToken) {
```

Comment:
Might want to explicitly handle the case where the website id doesn't exist. And possibly also the case where we're setting these settings on the default website instead of the specific one.